babel/lists.py

```diff
@@ -23,13 +23,14 @@
 if TYPE_CHECKING:
     from typing_extensions import Literal
 
-DEFAULT_LOCALE = default_locale()
+# TODO(3.0): Remove this.
+DEFAULT_LOCALE = default_locale()  # Please do not use this, it will be removed.
```

Comment:
Probably for another PR, but do we want to raise a DeprecationWarning for this attribute? That might dissuade people from using it.